test/transformers/test_cross_entropy.py

```diff
@@ -194,7 +194,7 @@ def _full_pass_once(B, T, V):
     ],
 )
 @pytest.mark.skipif(
-    torch.cuda.get_device_properties(0).total_memory < 64000000000,
+    torch.cuda.get_device_properties(0).total_memory < 64 * 1000 * 1000 * 1000,
```

Comment:
If readability is the concern, you can also use the underscore separator to delineate large numbers:

```
>>> 64_000_000_000 == 64 * 1000 * 1000 * 1000
True
```

Obviously just subjective though.

Reply:
TIL!